src/test/validation/PromotionQueryValidation.test.ts

```diff
@@ -275,6 +276,18 @@ describe('Unit tests for PromotionQueryValidation', function () {
     }
   });
 
+  test('Should fail if userId is not uuid', async () => {
+    try {
+      promotionQueryDTO.dayOfWeek = Day.MONDAY;
```

Comment:
Did we mean to change this line to promotionQueryDTO.userId equals some random non-uuid?

Reply:
Ohya good point, that's strange I have no idea why the test passed. I'll take a look thanks!

Reply:
Ah I realized it's because I didn't insert the `fail("Should have failed")` right after calling the validation